src/API/useFetch.tsx

```diff
@@ -1,20 +1,27 @@
 //@ts-nocheck
-import React, { useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import axios from "axios";
 const GITHUB_BASE_URL = 'https://api.github.com';
 
+export interface repositoryCommitMessage  {
+  date: {}
+  commitMessage: "string"
+  url: "string",
+  username: "string"
```

Comment:
This snytax looks off to me. Should it not just be the word `string` without the `""`?  Right now you are just setting the username type you the string "string" if that makes sense.

Reply:
you're right!